THIRD-PARTY-NOTICES

```diff
@@ -39,3 +39,49 @@ TASK FORCE DISCLAIMS ALL WARRANTIES, EXPRESS OR IMPLIED, INCLUDING
 BUT NOT LIMITED TO ANY WARRANTY THAT THE USE OF THE INFORMATION
 HEREIN WILL NOT INFRINGE ANY RIGHTS OR ANY IMPLIED WARRANTIES OF
 MERCHANTABILITY OR FITNESS FOR A PARTICULAR PURPOSE.
+
+License notice for The LLVM Compiler Infrastructure
+---------------------------
```

Comment:
Nit - could you please make the line of `----` to be the same length as the previous line?